fenix/app/src/main/java/org/mozilla/fenix/experiments/NimbusSetup.kt

```diff
@@ -102,8 +102,13 @@ fun NimbusInterface.maybeFetchExperiments(
     feature: NimbusSystem = FxNimbus.features.nimbusSystem.value(),
```

Comment:
Outside scope of issue - noticed the description of this func should reference the messaging fml - we could use this an opportunity to update the reference